build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
 }
 
 plugins {
-    id "org.jetbrains.kotlin.jvm" version "1.2.50"
+    id "org.jetbrains.kotlin.jvm" version "1.3.50"
```

Comment:
Needed to make contracts work (currently using them with `checkNotNull` function in order to let the compiler to smart cast the nullable type to the non nullable one in some parts of the code)